awscli/examples/networkflowmonitor/stop-query-monitor-top-contributors.rst

```diff
@@ -0,0 +1,9 @@
+**To stop a query**
+
+The following ``stop-query-monitor-top-contributors`` example stops the query in the specified account. ::
+
+    aws networkflowmonitor stop-query-monitor-top-contributors \
+        --monitor-name Demo \
+        --query-id aecd3a88-0283-35b0-a17d-6e944dc8531d
+
+This command produces no output.
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link